docs/release-notes.md

```diff
@@ -1,5 +1,35 @@
 # Release notes
 
+## Trino Gateway 14 (xx Jan 2025) { id="14" }
+
+Artifacts:
+
+* [JAR file gateway-ha-14-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/14/gateway-ha-14-jar-with-dependencies.jar)
+* Container image `trinodb/trino-gateway:14`
+* Source code as
+  [tar.gz](https://github.com/trinodb/trino-gateway/archive/refs/tags/14.tar.gz)
+  or [zip](https://github.com/trinodb/trino-gateway/archive/refs/tags/14.zip)
+* [Trino Helm chart](https://trinodb.github.io/charts/) `trino/trino-gateway` version `1.14.0`
+
+Changes:
+
+* Add support for automatic database schema validation and upgrade during Trino Gateway startup.
+  ([#575](https://github.com/trinodb/trino-gateway/pull/575))
+* Add support to configure HTTP client options for the external routing group selector. 
+  ([#580](https://github.com/trinodb/trino-gateway/pull/580))
+* Enable use of Oracle as backend RDBMS for Trino Gateway. 
```

Comment:
> as backend RDBMS

Why not just `backend database`?

Reply:
well.. its a type of database server .. the specific database instance is what you end up using .. but maybe to keep it simple I should stick to database.. will change